ui: Jobs / Statements description tooltip

vladlos opened the pull request:
Updated job description tool tip to truncate at around ~425 characters
Updated tooltip to 500px wide

Resolves: #46078

Release justification: bug fixes and low-risk updates to new functionality

Release note (ui): tooltips showing statements and jobs are limited in size for very long statements

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/46557)
<!-- Reviewable:end -->

dhartunian reviewed (changes requested):
@elkmaster can you add before/after screenshots to the change?

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/46557#-:-M3IojJSCAk3rufIycRU:b-81z63n)* status: :shipit: complete! 0 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

elkmaster commented:
### Before:
![Before jobs tooltip](https://user-images.githubusercontent.com/12850886/77656352-663b1f80-6f7c-11ea-9bd3-e6b75dc42f95.png)


### After:
**Statements tooltip**

![Statements tooltip](https://user-images.githubusercontent.com/12850886/77656374-70f5b480-6f7c-11ea-82b2-4dbe9d55eb23.png)

**Jobs tooltip**

![Jobs tooltip](https://user-images.githubusercontent.com/12850886/77656388-78b55900-6f7c-11ea-95a6-4015ddb3a927.png)

dhartunian reviewed (changes requested):
@elkmaster Looks great

Can you please add a release note: "tooltips showing statements and jobs are limited in size for very long statements"

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/46557#-:-M3hbDvL9tyAUSABAIqQ:bhnj7cx)* status: :shipit: complete! 0 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

vladlos committed:
ui: Jobs / Statements description tooltip

Updated job description tool tip to truncate at around ~425 characters
Updated tooltip to 500px wide

Resolves: #46078

Release justification: bug fixes and low-risk updates to new functionality

Release note (ui): tooltips showing statements and jobs are limited in size for very long statements

dhartunian reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> thanks!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/46557#-:-M3lFQjjDXBm795nzYgz:brfqlux)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

dhartunian commented:
bors r+
